src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -928,8 +927,16 @@ protected DynamoModel(IStartConfiguration config)
 
             CustomNodeManager = new CustomNodeManager(NodeFactory, MigrationManager, LibraryServices);
 
-            LuceneSearchUtility = new LuceneSearchUtility(this);
-            LuceneSearchUtility.InitializeLuceneConfig(LuceneConfig.NodesIndexingDirectory);
+            if (IsTestMode)
+            {
+                LuceneSearchUtility = new LuceneSearchUtility(this);
```

Comment:
I think the  LuceneSearchUtility = new LuceneSearchUtility(this); line should be outside of the if sentence (line 929),